extensions/wikia/ArticleComments/classes/ArticleCommentsAjax.class.php

```diff
@@ -169,9 +169,16 @@ static public function axReply() {
 	static public function axPost() {
 		global $wgRequest, $wgUser, $wgLang;
 
+		$result = [ 'error' => 1 ];
+
+		try {
+			$wgRequest->isValidWriteRequest( $wgUser );
+		} catch ( \BadRequestException $bre ) {
+			return $result;
```

Comment:
For better UI there should be an error message displayed for user with info what happened and what to do now. There is already translation key that can be reused: ```sessionfailure```